angular/projects/admin/src/app/services/document.service.ts

```diff
@@ -98,8 +98,15 @@ export class DocumentService {
       .valueChanges();
   }
 
-  query(documentTypeId: string, status: string, queryOpts: DocumentTypeQueryOptions = {}): Observable<Document[]> {
-    console.log(`[DocumentService:getDocumentTypeFields] ${documentTypeId}, query=${JSON.stringify(queryOpts)}`);
+  query(
+    documentTypeId: string,
+    status: string,
```

Comment:
Document status `'all'` doesn't exist.

Make status part of the `DocumentTypeQueryOptions` so that we don't need to pass in a non-standard `'all'` string for all queries that are *not filtering*.

Reply:
```typescript
export interface DocumentTypeQueryOptions {
  // ... 
  status: DocumentStatus;
  // ...
}
```

Reply:
Resolved on this commit https://github.com/oddbit/tanam/pull/220/commits/11cbc5687b780dba355845f7402967bed916a2d6